usecases/cluster/mocks/node_selector.go

```diff
@@ -0,0 +1,54 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package mocks
+
+import "sort"
+
+type memberlist struct {
+	// nodes include the node names only
+	nodes []string
+}
+
+func (m memberlist) StorageCandidates() []string {
+	sort.Strings(m.nodes)
+	return m.nodes
+}
+
+func (m memberlist) NonStorageNodes() []string {
+	return []string{}
+}
+
+func (m memberlist) SortCandidates(nodes []string) []string {
+	sort.Strings(m.nodes)
+	return nodes
+}
+
+func (m memberlist) NodeHostname(name string) (string, bool) {
+	for _, node := range m.nodes {
+		if node == name {
+			return name, true
+		}
+	}
+	return "", false
+}
+
+func (m memberlist) LocalName() string {
+	if len(m.nodes) == 0 {
+		return ""
+	}
+
+	return m.nodes[0]
```

Comment:
are we sure that `LocalName` always defaults to the first element of `nodes` list?

Reply:
in the mocks, given the most cases it's 1s node